Cargo.toml

```diff
@@ -45,6 +50,7 @@ test-support = { path = "crates/test-support" }
 textwrap = "0.11.0"
 which = "2.0.1"
 dirs = "1.0.4"
+volta-migrate = { path = "crates/volta-migrate", optional = true }
```

Comment:
`volta-migrate` is the name of both the executable in the main app (in `src/volta-migrate.rs`) and the library crate that holds the bulk of the migration code (in `crates/volta-migrate/`). This is referring the latter, which we need to have as a dependency so that the former can call into it.